Packs/KELARaDark/Integrations/RaDark/RaDark.py

```diff
@@ -0,0 +1,610 @@
+            alert.pop('title', None)
+
+            # Add monitor ID to the incident.
+            alert['monitor_id'] = MONITOR_ID
+
+            # Parse incident
+            incident = {
+                'name': get_name(alert['type']),
+                'occurred': timestamp_to_datestring(incident_created_time * 1000),  # timestamp in ms
+                'rawJSON': json.dumps(alert)
+            }
+            incidents.append(incident)
+
+        # Update last run if needed
+        if incident_created_time > latest_created_time:
+            latest_created_time = incident_created_time
+
+    # Save the next_run as a dict with the last_fetch key to be stored
+    next_run = {'last_fetch': latest_created_time}
+    return next_run, incidents
+
+
+def incident_get_items_command(client: Client, args: Dict[str, Any]) -> Union[CommandResults, None]:
+    try:
+        incident_id = str(args.get('incident_id', ''))
+        incident_data = client.incident_get_items(incident_id=incident_id)
+        if isinstance(incident_data, dict) and isinstance(incident_data.get('data'), dict) and \
+                isinstance(incident_data['data'].get('incidents'), list) and len(incident_data['data']['incidents']) > 0:
+            parsed_data, readable_output = parse_incident_markdown_table(incident_data['data'], incident_id)
+        else:
+            readable_output = f'No data found for item ID: {incident_id}'
+            parsed_data = []
+        return CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='Radark.itemDetails',
+            outputs_key_field='items',
+            outputs={'items': parsed_data, "details": readable_output})
+    except Exception as e:
+        demisto.error(traceback.format_exc())  # print the traceback
+        return_error(f'Failed to execute command.\nError:\n {str(e)} {incident_data}')
+        return None
+
+
+def email_enrich_command(client: Client, args: Dict[str, Any]) -> Union[CommandResults, None]:
+    try:
+        email = str(args.get('email', ''))
+        email_data = client.email_enrich_data(email=email)
+        if isinstance(email_data, dict) and isinstance(email_data.get('data'), dict) and \
+                isinstance(email_data['data'].get('incidents'), list) and len(email_data['data']['incidents']) > 0:
+            parsed_data, readable_output = parse_email_enrichment_markdown_table(email_data['data'])
+        else:
+            readable_output = f'No data found for email: {email}'
+            parsed_data = None
+        return CommandResults(
+            readable_output=readable_output,
+            outputs_prefix='Radark.emailDetails',
+            outputs_key_field='emails',
+            outputs={'emails': parsed_data})
+    except Exception as e:
+        demisto.error(traceback.format_exc())  # print the traceback
+        return_error(f'Failed to execute command.\nError:\n {str(e)} {email}')
+        return None
```

Comment:
It is unnecessary as the exception will be caught in the main function.
```suggestion
        raise Exception(f'RaDark Error: Failed to execute command.\nError:\n {str(e)} {email}')
```